programs/launcher/main.cpp

```diff
@@ -763,7 +763,7 @@ launcher_def::deploy_config_files (tn_node_def &node) {
                << " " << strerror(ec.value()) << endl;
           exit (-1);
         }
-        count = bf::remove_all (dd / "blockchain", ec);
+        count = bf::remove_all (dd / "shared_mem", ec);
         if (ec.value() != 0) {
```

Comment:
Line 760 has "blocks" and line 766 has "shared_mem". Both of these values are configurable via `config.ini`. Not sure why this is changed to "shared_mem". You removed the generation of this value in the `conifg.ini`. Seem like these values should be put in constants and used in the generation of `config.ini` and in the removal here. Otherwise there will be no way to know the values.

Reply:
The shared memory directory is no longer configurable on the eos-noon branch (and was renamed from a default of "blockchain" to a constant of "shared_mem"). I was just trying to clean this up.

Reply:
The issue with "blocks" is a real one (the user can configure this, but the launcher doesn't use the value when using -f).  I will either write a separate JIRA issue for this or roll it in with the next set of launcher changes I am doing.